components/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/connector/Attribute.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please don't update the license header year when you do a file change.

Reply:
fixed with https://github.com/wso2/carbon-identity-mgt/pull/182